introduction.md

```diff
@@ -3,3 +3,9 @@ Introduction
 May I introduce myself, the deadliest viper in the
```

Comment:
Since you're becoming a GitHub master, could you also add "Killing history using git rebase" as one of your deadly skills?